### DISCUSSION TITLE
Question 11 (quick): Does anyone use the Task tab in the various modules and what is its functionality? [MvK, DL, YR, KH, WG - Botany]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#980] [Q&A]
Description:  We have been thinking of using it, and tried in the loans module, for example, but as of this time of writing, the assigned people have not received an email notification - is that part of its functionality presumably?

**Comment by rondlg** [comment]
Response: 

Both Anthropology and Insects have dabbled with Tasks.

- EMu Notifications emails – 
- - Linked Parties record to Notify – must have Email in AddEmail field
- - Notification might not send if it is set after ~12p for the following date
- - Re-try with a notification date at least 2 days out
- - Emails often end up in spam

**Comment by fmjjones** [comment]
We have confirmation that Task notifications have been sent from EMu since the last client update, so if you aren't getting them submit a helpdesk ticket with the irn of the record with the task and the name of the Module.
